### DISCUSSION TITLE
Checking if Coloring file exists.. Segmentation fault

### DISCUSSION THREAD

**Discussion by raisePiginYourhome** [mdolab/dafoam#455] [General]
when Running Coloring Solver, console posts this error. How can I solve this or some advices?
thanks!!!!!!!

**Comment by friedenhe** [comment]
We can't debug with the limited information you provided. Please post the entire log file and elaborate on what you did before getting this error.

**Reply by raisePiginYourhome** [reply]
it has been solved, thank you.

**Reply by alinaeem121212** [reply]
Could you please guide me on how you resolved this error?
